nengo/builder/operator.py

```diff
@@ -615,24 +615,12 @@ def step_dotinc():
 
 
 class SparseDotInc(DotInc):
-    """Like `.DotInc` but ``A`` is a sparse matrix.
-
-    This class only overrides ``make_step``. See `.DotInc` for other details.
-    """
+    """Like `.DotInc` but ``A`` is a sparse matrix."""
 
     def __init__(self, *args, **kwargs):
+        # Disallow reshaping
         super().__init__(*args, reshape=False, **kwargs)
```

Comment:
Should we explicitly check that `A` is sparse here?

Reply:
Sure, no reason not to.